### PR TITLE
Polishes version header documentation

### PR DESCRIPTION
**Pull request by vulder**
related se-passau/VaRA#571

### CODE REVIEW

File: varats/data/version_header.py [hunk 1]
```diff
@@ -70,13 +78,21 @@ def doc_type(self) -> str:
         return self.__doc_type
 
     def is_type(self, type_name: str) -> bool:
-        """Checks if the type of the following yaml file is type_name."""
+        """
+        Checks if the type of the following yaml file is type_name.
```
Comment (boehmseb) [inline varats/data/version_header.py]: `type_name` in double-backticks?

File: varats/data/version_header.py [hunk 2]
```diff
@@ -60,6 +64,10 @@ def from_version_number(cls, doc_type: str,
         """
         Creates a new VersionHeader object from a doc_type string and a
```
Comment (boehmseb) [inline varats/data/version_header.py]: `doc_type` in double-backticks?

**Review by boehmseb — APPROVED** [review]

File: varats/data/version_header.py (hunk 2 above)
Comment (vulder) [inline varats/data/version_header.py]:
```suggestion
        Creates a new VersionHeader object from a ``doc_type`` string and a
```

File: varats/data/version_header.py (hunk 1 above)
Comment (vulder) [inline varats/data/version_header.py]:
```suggestion
        Checks if the type of the following yaml file is ``type_name``.
```

**Review by boehmseb — APPROVED** [review]

**Pull request merged**